### DISCUSSION TITLE
Citations

### DISCUSSION THREAD

**Discussion by jpitbull917** [SnailyCAD/snaily-cadv4#1444] [Q&A] [answered]
How do I accept citations?

**Comment by casperiv0** [comment]
Can you elaborate?

**Reply by jpitbull917** [reply]
Like when I write tickets, it don't show when I search the citizen.

**Reply by casperiv0** [reply]
If your CAD has `Citizen records approval` enabled, the tickets, arrest reports and written warnings must first be accepted via Officer -> Citizen Logs -> Pending Arrest Reports

If you don't see the records created, update your CAD since a bug fix was just pushed for this.

**Comment by casperiv0** [comment, marked as answer]
If your CAD has `Citizen records approval` enabled, the tickets, arrest reports and written warnings must first be accepted via Officer -> Citizen Logs -> Pending Arrest Reports

If you don't see the records created, update your CAD since a bug fix was pushed for this.
